lib/main.dart

```diff
@@ -230,6 +235,9 @@ class PendingTasksPage extends StatelessWidget {
   Widget build(BuildContext context) {
     var appState = context.watch<MyAppState>(); // Get the app state
 
```

Comment:
I really like the look on Iphone but I think the look on the navigator is a bit poor and can be improved a lot.